src/components/account/details/FormField.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react'
+
+import '../../../styles/account/details/form_field.css'
+
+export default class EditAccountDetails extends Component {
+  render() {
+    const { label, value } = this.props
+    return (
+      <div className="form--read-only">
```

Comment:
This is part of BEM again... double underscore___ separates blocks from elements...and double hyphen -- separates modifiers from elements:
http://getbem.com/naming/